src/main/java/com/example/demo/entities/ImageEntity.java

```diff
@@ -0,0 +1,15 @@
+package com.example.demo.entities;
+
+import jakarta.persistence.*;
```

Comment:
wildcard import

Reply:
IJ IDEA automatically imports wildcard when using @Lob. Seems to be it's bug I haven't noticed